src/Maestro/Maestro.Data/Models/Build.cs

```diff
@@ -163,4 +163,12 @@ public class BuildDependency
         // To this build's repository and branch
         public double TimeToInclusionInMinutes { get; set; }
     }
+
+    public class GoalTime
+    {
+        public int ChannelId { get; set; }
+        public int DefinitionId { get; set; }
+        public int Minutes { get; set; }
+        public Channel Channel { get; set; }
```

Comment:
>Channel [](start = 23, length = 7)

Please move this property to be just below the ChannelId property. The Foreign Key and its Navigation Property should be co-located.

Reply:
added channel under channelID